### PR TITLE
Fixes #16775: rl add an optional path for compilation and translation

### PR DESCRIPTION
**Pull request by gpoblon**
https://issues.rudder.io/issues/16775

### CODE REVIEW

**Comment by gpoblon** [comment]
Work done, ready to be reviewed

**Comment by peckpeck** [comment]
One commit is still a wip here

**Comment by gpoblon** [comment]
rebased, shoud be mergeable

**Comment by gpoblon** [comment]
PR rebased

File: rudder-lang/tests/test_files/tester/rl/simplest.rl [hunk 1]
```diff
@@ -10,5 +10,5 @@ resource simplest()
 
 simplest state technique() {
   @component = "File absent"
-  if ms_dos => file("tmp").absent() as file_absent_tmp
```
Comment (amousset) [inline rudder-lang/tests/test_files/tester/rl/simplest.rl]: dropping ms-dos support? :cry:

Comment (gpoblon) [reply rudder-lang/tests/test_files/tester/rl/simplest.rl]: gotta pressure the big boi

**Pull request merged**